SharpPcap/WinDivert/WinDivertDevice.cs

```diff
@@ -206,7 +206,7 @@ public void SendPacket(ReadOnlySpan<byte> p)
         /// Thread that is performing the background packet capture
         /// </summary>
         protected Thread captureThread;
-        private bool shouldCaptureThreadStop;
```

Comment:
Can this commit be fixed up with the first commit as this would seem to overlap entirely with the changes in that commit?